data-8/datascience
# Address formatting problems from matplotlib 2.x upgrade

[ ] Wrote test for feature (*Couldn't due to lack of GUI tests*)
[x] Added changes in the Changelog section in README.md
[x] Bumped version number (delete if unneeded)

## Changes proposed:
Addresses some minor formatting issues and warnings introduced by upgrading to matplotlib 2.x. See https://github.com/data-8/datascience/issues/324 for details.

## To test:

1. Generate a histogram. Before (i.e., on `f73aaf45f8b3`), you'll see a warning about the `normed` keyword and no lines around the bars; both these issues should be fixed after.

2. Generate a bar chart. Before (i.e., on `f73aaf45f8b3`), you'll see the labels are offset from the bars, but this issue should be fixed after.
